content-services/latest/admin/cluster.md

```diff
@@ -332,9 +333,58 @@ This topic describes the instructions for installing and configuring Solr nodes
 5. If you're using HTTP transport, then make sure that the following property is set in `<classpathRoot>/alfresco-global.properties`:
 
     ```text
-    `solr.secureComms=none`
+    solr.secureComms=none
     ```
 
+6. If you wish to proceed with the default embedded Hazelcast clustering mechanism, you're done.
+
+    However, if you wish to manage your own external Hazelcast cluster, you should additionally follow the instructions highlighted in the next section [Set up repository clustering via external Hazelcast](#set-up-repository-clustering-via-external-hazelcast).
+
+### Set up repository clustering via external Hazelcast
+
+Managing your own Hazelcast cluster allows you to scale repository and Hazelcast instances independently, while also allowing you to manage resources allocation on an individual level. This kind of setup provides more fine-grained control at the cost of more necessary preliminary work.
+
+1. Prepare a valid Hazelcast XML configuration file including all caches definitions required by the repository. This can be mostly auto-generated for you by doing the following:
+
+    1. Locate the `caches.properties` file within the `alfresco.war:WEB-INF\lib\alfresco-repository-*.jar` archive.
+    2. Copy the `caches.properties` file to an easily accessible directory.
+    3. Clone the [alfresco-enterprise-repo.git](https://github.com/Alfresco/alfresco-enterprise-repo.git){:target="_blank"} repository locally.
```

Comment:
```suggestion
    3. Clone the [alfresco-community-repo.git](https://github.com/Alfresco/alfresco-community-repo.git){:target="_blank"} repository locally.
```

Reply:
The python script has been moved to community-repo: https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/scripts/hazelcast-init/generate-hazelcast-config.py.